STORM-IRIT/scripts/run.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -ex
+
+process(){
+    base_name=$(basename -- "$1")
+    filename="${base_name%.*}"
+    path=output/$filename
+    
+    python3 preprocess_cloud.py $base_name
+    ./pdpcComputeMultiScaleFeatures -i "${filename}.ply" -v -o $path/$filename
+    ./pdpcSegmentation -i "${filename}.ply" -s $path/"${filename}_scales.txt" -f $path/"${filename}_features.txt" -o $path/$filename -v
+
+    ./pdpcPostProcess -i "${filename}.ply" -s $path/"${filename}_seg.txt" -c $path/"${filename}_comp.txt" -o $path/"${filename}_range" -col -v -range 20 24 25 30 40 42
+
+    ./pdpcPostProcess -i "${filename}.ply" -s $path/"${filename}_seg.txt" -c $path/"${filename}_comp.txt" -o $path/"${filename}_pers" -col -v -pers 15 20 25
+
+    ./pdpcPostProcess -i "${filename}.ply" -s $path/"${filename}_seg.txt" -c $path/"${filename}_comp.txt" -o $path/"${filename}_scales" -col -v -scales 5 15 20 25
+    rm $path/"${filename}_scales.txt"
+    rm $path/"${filename}_features.txt"
+    rm $path/"${filename}_seg.txt"
+    rm $path/"${filename}_comp.txt"
+    for f in $path/*.ply; do
```

Comment:
As I understand it will go through dirs and look for all files, but what about flattening? Looks like the result will be saved in the dir `scale`, `range` or so on. Maybe I skip smth?

Reply:
These folders are not created and saving paths are modified, so it will be flattened